libusb/os/linux_usbfs.c

```diff
@@ -182,7 +182,7 @@ static int dev_has_config0(struct libusb_device *dev)
 	return 0;
 }
 
-static int get_usbfs_fd(struct libusb_device *dev, mode_t mode, int silent)
+static int get_usbfs_fd(struct libusb_device *dev, int mode, int silent)
```

Comment:
this one I'd actually call a step back in readability in general

Reply:
maybe a `typedef` for readability?

Reply:
Well, whereas for the other commits it is a lot of sprinkling on casts, this one actually corrects a wrong type. The type should be int and not mode_t. Did you see the commit message? Would you prefer a better variable name?

Reply:
> Did you see the commit message?

I must confess - I didn't (now I did).

> Would you prefer a better variable name?

Maybe a `open_flags`? That would be as straight-forward as possible, since it is getting passed into `open` as `flags` argument.

Reply:
To follow the terminology of the open() manual page I think `access_mode` is a good choice. It is only used to carry the access mode here, and no other flags.

Reply:
sure, sounds good